.github/workflows/examples.yml

```
@@ -90,10 +100,10 @@ jobs:
           file: bin/${{ matrix.component.file }}
           oci-reference-without-tag: ghcr.io/${{ github.repository_owner }}/${{ matrix.component.name }}
           version: ${{ steps.meta.outputs.tag }}
-      - name: Publish ${{ matrix.component.name }} latest (if latest)
+      - name: Publish ${{ matrix.component.name }} with special tag (if specified)
```

Nit: Could you change this to be:
 
```suggestion
      - name: Publish ${{ matrix.component.name }} with tag ${{ steps.meta.outputs.release_tag || 'latest' }} (if specified)
```